Classes/Settings.cs

```diff
@@ -490,5 +481,32 @@ protected static void SetValue(string propertyName, string value)
                 }
             }
         }
+
+        private static string GetValuePostfix(Model.Programme progInfo, string name, string defaultValue)
+        {
+            string value = null;
+            if (progInfo != null)
+            {
+                value = GetValue(name + progInfo.Progid.ToString(CultureInfo.InvariantCulture));
```

Comment:
Ah, yeah - that can be the issue with editing comments - people don't get notified.

I think I still prefer the idea of a separate programme_settings table.  Without actually sitting down and writing it, it seems like it should involve cleaner code than the other options.